build/scripts/build-bazel-s390x.sh

```diff
@@ -0,0 +1,276 @@
+#!/bin/bash
+# © Copyright IBM Corporation 2024
+#
+# Use of this source code is governed by a LICENSE_NAME-style
+# license that can be found in licenses/APACHE-v2.0.txt.
+# LICENSE: Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+#
+# Portions of this file are additionally subject to the following
+# license and copyright.
+#
+# Copyright 2025 The Cockroach Authors.
+#
+# Use of this software is governed by the CockroachDB Software License
+# included in the /LICENSE file.
+#
+# This code originated in https://github.com/linux-on-ibm-z/scripts/blob/master/CockroachDB/24.1.6/build_crdb.sh, SHA d2d885b3b9edae9d0e69383c18475485149875b1
+#
```

Comment:
I'm suggesting leaving a blank line between the license header and the rest of the file to clearly delineate the license header comment from the first non-license comment in the file. I do see this is a change from the original file.

```suggestion

```